css/style.css

```diff
@@ -237,7 +237,9 @@ hr {
 .about-content {
     margin-top: 50px;
 }
-
+.backtotop{
```

Comment:
There is no element with this `.backtotop` class in the html file, and there is already `scroll-behavior: smooth;` present for `*` property so there is no need for it again.